### DISCUSSION TITLE
IEA-15MW-240 Free Decay Tests

### DISCUSSION THREAD

**Discussion by Uisliu** [OpenFAST/openfast#2352] [Q&A]
Hi all, I’ve been attempting to run free decay tests on the IEA-15MW-240-RWT. Having run tests and comparing results to the IEA-15MW-240-RWT documentation, My OpenFAST results seem to be a bit off. I’ve searched the NREL forums and have made the alterations found down below. And yet they seem just a little off. I also saw that discrepancies in my results could be due to the alterations made to the mass matrix. Below are the changes I made to my simulations.

GenDOF=Off
PCMODE=0
VSContro=0
RotSpeed = 0
CompInflow = 0
CompAero = 0
GenTiStr = True
TimeGenOn .> Tmax

The appropriate initial conditions. I.e. initial platform displacements.

Set the RNA Offset to zero by altering the following settings in Elastodyn
Precone angles=0
Overhang = 0
NacMxn = 0
NacMzn = 0

The following are the results I got compared to the Documentation. The Documentation points on the graphs below represent the high and low peaks of the graphs in the IEA-15MW-240 definition. 
 
 
![image](https://github.com/user-attachments/assets/d7073af5-5c43-42cc-83cb-2de1bd46a8ff)

  
![image](https://github.com/user-attachments/assets/29dbbcf7-8716-4c63-a5b6-121c3a123178)

![image](https://github.com/user-attachments/assets/4c3f76d5-9a92-4e54-83e2-d769edeed76e)

![image](https://github.com/user-attachments/assets/d4a5edc9-7bbb-49eb-b07a-f9ff2d564f7c)

 
![image](https://github.com/user-attachments/assets/22f22748-e44b-45e5-bc3b-f6337e232af1)

 
![image](https://github.com/user-attachments/assets/484d20e7-c3d5-4156-989f-f89940e6a279)

 

Are there any obvious issues that I’m making with the tests to get these results. I saw on other posts that others were getting closer results and so I just want to make sure Im doing things right
Thank you,
Uisliu

**Comment by jjonkman** [comment]
Dear @Uisliu,

I was not involved in the development of the UMaine semisubmersible for the IEA Wind 15-MW RWT and don't know why you results are slightly different than what is referenced in the specification document.  I asked @mattEhall and @gbarter who were coauthors of that report, but they were not the ones to run these free-decay simulations either.  It could be that your simulation set up is a bit different from the original or that the model evolved a bit since that report was published.

Regardless, your results are fairly close.

Best regards,

**Reply by Uisliu** [reply]
Hi @jjonkman 

Thank you so much for your reply and your input. It is much appreciated! I'm happy with my results in other areas so these being close enough is fine by me.

Thank you so much again,

best regards!
